errors.go

```diff
@@ -146,7 +146,7 @@ func (e NotAcceptableError) Unwrap() error { return HTTPError(e) }
 
 // ErrorHandler is the default error handler used by the framework.
 // If the error is an [HTTPError] that error is returned.
-// If the error adheres to the [ErrorWithStatus] and/or [ErrorWithDetail] interface
+// If the error adheres to the [ErrorWithStatus] interface
```

Comment:
I see that this was a comment mistake. However should we transform errorwithdetail as well?

Reply:
Mmmh I don't think so. 
Maybe more ErrorWithTitle than ErrorWithDetail anyway.